Library/OcMainLib/OpenCoreNvram.c

```diff
@@ -203,12 +203,117 @@ OcAddNvram (
   }
 }
 
+#if defined (MDE_CPU_X64)
+// test rax, rax
+// jz ...
+STATIC CONST UINT8  CheckSecurityListGetSet[] = {
+  0x48, 0x85, 0xC0, 0x74
+};
+
+// xor rax, rax
+// jz ...
+STATIC CONST UINT8  DontCheckSecurityListGetSet[] = {
+  0x48, 0x31, 0xC0, 0x74
+};
+
+// test rbx, rbx
+// jz ...
+STATIC CONST UINT8  CheckSecurityListGetNext[] = {
+  0x48, 0x85, 0xDB, 0x74
+};
+
+// xor rbx, rbx
+// jz ...
+STATIC CONST UINT8  DontCheckSecurityListGetNext[] = {
+  0x48, 0x31, 0xDB, 0x74
+};
+#endif
```

Comment:
@PMheart commented that these might be too short.

My reply was that I am replacing within short functions (see search lengths below) and I know that I am  searching exactly from the start of the function. Therefore I thought these were likely to be specific enough, and more likely to work on variants of the functions. I could add more bytes, but thought it might link them to too specific a version of the functions, since it has to work on firmware for several different Macs.

As I said (in a lost comment, due to force push not within PR - it's now a PR!), I am not certain of the best approach, and I wondered if @vit9696 (or @khronokernel ??) or anyone else had any time to confirm if this looks okay or not? Many thanks in advance if so.

Reply:
Btw, the basic structure of the functions is:
 - get a pointer to a linked list
 - if not NULL, loop round the linked list calling installed security check methods, aborting if check fails
 - do the thing we want
So I'm just forcing the early check to treat the list of security functions as empty.

Reply:
(Probably should add, I think I'll find out more if this current version fails or crashes on some macs, rather than just failing to match as would become more likely if I add more.)